src/components/domain/Benefit/index.jsx

```diff
@@ -0,0 +1,31 @@
+import { ContentContainer } from '@components/base';
+import { ContentItem } from '@components/base';
+import { Toggle } from '@components/base';
+import { BORDER_STYLE } from '@utils/constants';
+import styled from 'styled-components';
+
+const Others = () => {
+  return (
+    <Wrap>
+      <ContentContainer title="상품 혜택 허용 설정" />
+      <ContentItem title="마일리지 적립">
+        <StyledToggle>
+          <Toggle on />
+        </StyledToggle>
+      </ContentItem>
+    </Wrap>
```

Comment:
이거 이렇게 사용하시면 돼요! Wrapper 없이 쓰실 수 있게 만들어뒀습니당
```suggestion
      <ContentContainer title="상품 혜택 허용 설정">
        <ContentItem title="마일리지 적립">
          <StyledToggle>
            <Toggle on />
          </StyledToggle>
        </ContentItem>
      </ContentContainer>
```

Reply:
수정 완료했습니다!!